### PR TITLE
Fix TR-3287 remove source based npm dependencies

### PR DESCRIPTION
**Pull request by wazelin**
# [TR-3287](https://oat-sa.atlassian.net/browse/TR-3287)

- fix: remove source-based npm dependencies by updating `raphael` package dependency to `2.3.0`

### CODE REVIEW

**Review by yohanlaborda — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Pull request's target is not master
- [X] Commits are following conventional commits
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by poyuki — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
         "postcss-scss": "^2.0.0",
         "prettier": "^2.4.1",
         "qunit": "^2.17.2",
-        "raphael": "2.2.0",
+        "raphael": "^2.3.0",
```
Comment (btamas) [inline package.json:89]: It should be exactly 2.3.0
```suggestion
        "raphael": "2.3.0",
```

**Review by btamas — CHANGES_REQUESTED** [review]

**Comment by krampstudio** [comment]
merging since the change requested by Tamas has been addressed

**Comment by krampstudio** [comment]
It looks like one test is failing, but it is failing on develop too :/

**Pull request merged**